Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -122,6 +123,36 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
             this._fontVariationsGroup.rows = [emptyRow];
         }
+
+        // Custom Font Properties
+
+        // Display message when we have no custom fonts.
+        if (!this.nodeStyles.customFonts.length) {
+            this._customFontsRow.showEmptyMessage();
+            this._customFontsRow.dataGrid = null;
+            return;
```

Comment:
we typically dont add early `return` in functions like this as it makes it harder for new things to be added at the end in the future.  as such, id put everything after this in an `} else { ... }` (or more ideally flip it around and have the positive check be first with the error case in the `else`)